packages/backend/src/api/endpoints/OrganizationApi.ts

```diff
@@ -116,6 +121,34 @@ export class OrganizationAPI extends AbstractAPI {
     });
   }
 
+  public async updateOrganizationLogo(organizationId: string, params: UpdateLogoParams) {
+    this.requireId(organizationId);
+
+    if (params?.file === null) {
+      return this.request<Organization>({
+        method: 'DELETE',
+        path: joinPaths(basePath, organizationId, 'logo'),
+      });
+    }
+
+    if (!params?.file) {
+      throw new Error('A valid Blob or File is required.');
+    }
+    if (!params?.uploaderUserId) {
+      throw new Error('A valid uploaderUserId is required.');
+    }
```

Comment:
:upside_down_face: Feel free to let the Backend API do the validations for us. Not sure if we want fast feedback at least for non empty values.